src/elements/footer-nav.html

```diff
@@ -0,0 +1,103 @@
+<link rel="import" href="../../bower_components/polymer/polymer-element.html">
+<link rel="import" href="../../bower_components/iron-image/iron-image.html">
+
+
+<dom-module id="footer-nav">
+  <template>
+    <style is="custom-style" include="shared-styles flex flex-alignment"></style>
+
+    <style>
+
+      .nav-inline {
+        list-style: none;
+        margin: 0;
+        padding: 0;
+      }
+
+      .gdg-logo {
+        margin: 10px 30px 0 0;
+        height: 24px;
+        width: 120px;
+        float: left;
+      }
+
+      .copyright {
+        margin-right: 0;
+        padding: 15px 0 0 0;
+        box-sizing: border-box;
+        float: right;
+      }
+
+      .cod {
+        margin-right: 0;
+        padding: 15px 0 0 0;
+        float: left;
+      }
+
+      a {
+        -moz-osx-font-smoothing: grayscale;
+        -webkit-font-smoothing: antialiased;
+        border-bottom: 1px solid transparent;
+        color: #616161;
+        padding-bottom: 2px;
+        text-decoration: none;
+        transition: border-color .3s ease-out;
```

Comment:
Can we move animation global config to variables to make it the same everywhere?